brainscore_language/model_helpers/huggingface.py

```diff
@@ -115,6 +121,97 @@ def digest_text(self, text: Union[str, List[str]]) -> Dict[str, DataAssembly]:
             if self.neural_recordings:
                 representations = self.output_to_representations(layer_representations, stimuli_coords=stimuli_coords)
                 output['neural'].append(representations)
+        return output
+
+    def _masked_inference(self, text):
```

Comment:
there's a lot of duplicate code here (with `_causal_inference`), can we unify this?

Reply:
I agree. [a4c52f4](https://github.com/brain-score/language/pull/211/commits/a4c52f4f074e16b3520fbe435eea0bb34ee271a1) refactors a lot of the common code into separate methods.